src/client/settings/templates/settings-cache.html

```diff
@@ -14,7 +14,7 @@
 
         <md-input-container>
             <md-checkbox ng-model="settings.config.deleteTmpOnClose">
-                Clear Tmp Folder after closing app?
```

Comment:
If you talking about translations in commits I agree that it should not be inside commits (maybe a separate repo ?). But for english strings inside the code I don't see why not.